src/api/store/technology.py

```diff
@@ -352,6 +352,7 @@ def create_new_vendor_for_technology(self, context: Context, args) -> Tuple[Tech
         try:
             technology_id = args.pop('technology_id', None)
             user = get_user_from_context(context)
+            is_published = args.pop('is_published', True)
```

Comment:
why are we defaulting to true?
Shouldn't we check if its there then we set it.  if its not there then we dont set it?